libfastpm/cosmology.c

```diff
@@ -150,8 +151,10 @@ double D2Omega_ncdmTimesHubbleEaSqDa2(double a, FastPMCosmology * c)
 double HubbleEa(double a, FastPMCosmology * c)
 {
     /* H(a) / H0 
-       ncdm is NOT assumed to be matter like here */
-    return sqrt(Omega_r(c) / (a*a*a*a) + c->Omega_cdm / (a*a*a) + Omega_ncdmTimesHubbleEaSq(a, c) + c->Omega_Lambda);
```

Comment:
I guess the original comment "ncdm is NOT assumed to be matter like here" really means ncdm is not assumed to be "fully" matter like here? It is still a mixture of matter like and radiation like computed from the distribution function?

Reply:
Yeah, I can add the word "fully". With the distribution function, the split into two class "matter-like" and "radiation-like" isn't strictly true, which is why I didn't bother with "fully", but I agree it'll make things clearer.

Reply:
Yeah, I can add the word "fully". With the distribution function, the split into two class "matter-like" and "radiation-like" isn't strictly true, which is why I didn't initially write "fully", but I agree it'll make things clearer.